diff -urN OpenLieroX/src/breakpad/external/src/common/linux/file_id.cc OpenLieroX-patched/src/breakpad/external/src/common/linux/file_id.cc
--- OpenLieroX/src/breakpad/external/src/common/linux/file_id.cc	2010-01-26 06:56:11.000000000 +1000
+++ OpenLieroX-patched/src/breakpad/external/src/common/linux/file_id.cc	2011-12-14 10:57:51.110205156 +1100
@@ -40,6 +40,7 @@
 #include <link.h>
 #include <string.h>
 #include <sys/mman.h>
+#include <sys/stat.h>
 #include <unistd.h>
 
 #include <cassert>
